### DISCUSSION TITLE
Support for 8.12

### DISCUSSION THREAD

**Discussion by ksshivaprasad** [fbaligand/kibana-enhanced-table#344] [General]
I am running Kibana 8.12.2 version. May I know when we will get support for it? Thank you!

**Comment by fbaligand** [comment]
Well, it was already available here:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_8.12.2.zip

It has been published 6 days after Kibana 8.12.2 release.

**Reply by ksshivaprasad** [reply]
Excellent! Looks like I missed it. Thank you very much for your quick response!
